src/main/java/ui/components/pickers/PickerMilestone.java

```diff
@@ -97,6 +100,59 @@ private void setRemovedInUI(Label milestone) {
         milestone.getStyleClass().add("labels-removed"); // add strikethrough
     }
 
+    /**
+     * Gets the existing milestone from the milestoneList
+     *
+     * @param milestoneList
+     * @return Optional of existing milestone
+     */
+    public static Optional<PickerMilestone> getExistingMilestone(List<PickerMilestone> milestoneList) {
+        return milestoneList.stream()
+                .filter(PickerMilestone::isExisting)
+                .findAny();
+    }
+
+    /**
+     * Gets the selected milestone from the milestoneList
```

Comment:
@HansNewbie 
The should only be one existing and selected milestone at any point in time actually. 
Because each issue can only be assigned to one milestone.